CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
```

Comment:
I guess this is a nit, but should we put as the last bullet point in the section 'Examples of unacceptable behavior by participants include:'

* The use of sexualized language or imagery, except in discussion of site behavior concerning adult content, as specified immediately below

(or similar, feel free to reword for clarity)

The way it reads now we don't explicitly say 'This is *not* allowed except when'. Adding the above would remove that ambiguity.

With that one possible exception for clarity, I like this and I'm behind it.